apps/ideaspace/src/components/modules/DashboardPage/DashboardPage.js

```diff
@@ -2,46 +2,38 @@ import React from 'react'
 import axios from "axios";
 import Grid from '@mui/material/Grid';
 import { useUserDataContext } from '@devlaunchers/components/context/UserDataContext';
-import SignInButton from "../../common/SignInButton/SignInButton";
+import SignInSection from '../../common/SignInSection/SignInSection';
 import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader'
 import Stats from './Stats/Stats';
 import SortableDropdown from '../../common/SortableDropdown/SortableDropdown';
 import IdeaCard from './IdeaCard/IdeaCard'
 import submitImage from "../../../images/submitButton.svg";
 import Link from 'next/link';
 import RainbowBar from '../../../../../website/src/components/common/RainbowBar';
+import { atoms } from '@devlaunchers/components/src/components';
 
 import {
-  PageWrapper,
-  Title,
-  RainbowArea,
+  HeadWapper,
+  Headline,
   StyledRanbow,
-  Slogan,
-  SectionTitle,
+  PageWrapper,
   IdeaCardWrapper,
-  SorterDiv,
-  Button,
-  CardWapper,
-  Headline,
-  Description,
-  Container,
-  Wrapper
+  SubmitButton,
 } from './StyledDashboardPage';
 
 
 function DashboardPage() {
 
   let { userData, setUserData, isAuthenticated } = useUserDataContext();
   //console.log("dashpage", userData);
-  //if (process.env.NEXT_PUBLIC_NAME == "DEVELOPMENT") isAuthenticated = true;
+  if (process.env.NEXT_PUBLIC_NAME == "DEVELOPMENT") isAuthenticated = true;
```

Comment:
is this line necessary?

Reply:
It is convinience for developer, not necessary for production.